### PR TITLE
Refactory test pages

### PR DESCRIPTION
**Pull request by Hugo-Hontans**

### CODE REVIEW

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import NewTemplatePage from "./NewTemplatePage";
+
+export default class DeliveryFormPage extends NavbarPage {
+  constructor() {
+    super();
+    cy.get('.delivery-form').should('contain', 'Sign delivery');
+  }
+
+  public verifyDeletedMaterial(value: string) {
+    cy.get('mat-card').contains(value).should('have.length', 0);
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: `mat-card` is too generic, use class / test ids

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+      expect(footers).to.have.length(count) });
+  }
+
+  public clickSign() {
+    cy.get('[testId=modalSign]').find('button').contains('Sign').click();
+    cy.wait(3000);
+  }
+
+  public fillPassword(password: string) {
+    cy.get('input[type="password"]').type(password);
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: there's a password form in the delivery page?

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  }
+
+  public clickVerifyToSign() {
+    cy.get('[testId=sign]').click();
+  }
+
+  public clickAddSignature() {
+    cy.get('button').contains('Add signature').click();
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: this'll break when we change the copy (texts within the page), use a test id?

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  }
+
+  public clickDeleteMaterial(value: string) {
+    cy.wait(1000);
+    cy.get('mat-card')
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: too generic?

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    .contains(value)
+    .parent().parent()
+    .trigger('mouseover')
+    .find('button').contains('DELETE').click({force: true});
+  }
+
+  public clickCheckBoxMaterial(name: string) {
+    cy.get('mat-card')
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: too generic?

File: apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import { HomePage, OrganizationFormPage, LandingPage, ViewProfilePage, MovieTeamWorkPage, DeliveryTeamWorkPage } from ".";
+
+export default abstract class NavbarPage {
+  constructor() {
+    cy.get('.account-icon', {timeout: 60000}).contains('account_circle');
+  }
+
+  public assertEncryptingExists() {
```
Comment (lsenta) [inline apps/delivery/delivery-e2e/src/support/pages/NavbarPage.ts]: `assertIsEncrypting`

File: apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts (hunk 2 above)
Comment (Hugo-Hontans) [reply apps/delivery/delivery-e2e/src/support/pages/DeliveryFormPage.ts]: Yes for sign a delivery

File: .circleci/config.yml [hunk 7]
```diff
@@ -17,6 +17,7 @@ jobs:
             - dependencies-{{ .Environment.CACHE_VERSION }}-{{ checksum "package.json" }}
             # fallback to using the latest cache if no exact match is found
             - dependencies-{{ .Environment.CACHE_VERSION }}
+      - run: rm -rf node_modules/ganache-core/node_modules/web3-providers-ws/node_modules/websocket/.git
```
Comment (lsenta) [inline .circleci/config.yml]: Could you rename the commit message to be explicit?

`git rebase -i HEAD~8` => use `fixup` => change the commit message, save and quit

Comment (Hugo-Hontans) [reply .circleci/config.yml]: done

**Pull request merged**